src/Featurizers/CountVectorizerFeaturizer.h

```diff
@@ -26,15 +27,14 @@ class CountVectorizerTransformer : public StandardTransformer<std::string, Spars
     // |
     // ----------------------------------------------------------------------
     using BaseType                           = StandardTransformer<std::string, SparseVectorEncoding<std::uint32_t>>;
-    using IndexMapType                       = std::unordered_map<std::string, std::uint32_t>;
-    using AnalyzerMethod                     = Components::AnalyzerMethod;
-
+    using TfidfEstimator                     = Microsoft::Featurizer::Featurizers::TfidfVectorizerEstimator<std::numeric_limits<size_t>::max()>;
```

Comment:
>TfidfEstimator [](start = 10, length = 14)

It doesn't look like this type is used outside of the definition of TransformerUniquePtrType.